pyproject.toml

```diff
@@ -23,7 +23,7 @@ packages = [{ include = "qibotn", from = "src" }]
 python = "^3.9,<3.12"
 qibo = "^0.2.4"
 quimb = { version = "^1.6.0", extras = ["tensor"] }
-cupy = { version = "^11.6.0", optional = true }
+cupy-cuda12x = { version = "^11.6.0", optional = true }
 cuquantum-python-cu11 = { version = "^23.3.0", optional = true }
```

Comment:
As `copy-cuda12x` is a wheel package for CUDA 12, shall we also change `cuquantum-python-cu11` to `cuquantum-python-cu12` for consistency?

Reply:
Correct, I will change to `cupy-cuda11x`, assuming that it has been tested with CUDA 11.

Reply:
Fixed in a5c24a6